docs/modules/operation/pages/database-reports/templates/maintenance-expired.adoc

```diff
@@ -1,28 +1,56 @@
 [[maintenance-expired-report]]
 = Maintenance Contracts Expired Report
 
-Asset management report shows all maintenance contracts expired.
+Asset management report shows all assets with maintenance contracts that are expiring within a threshold period, which starts today and extends for the number of days specified in the Warning Threshold parameter below.  This report can also show assets with maintenance contracts that have already expired.
+
+The report includes a table that list assets that match the criteria defined by the parameters.  This table is sorted by node and then ordered by expiration date.  The report includes the node label, manufacturer, maintenance contract, expiration date and status.
+// QUESTION: It is not clear what is in the field "maintenance contract" in the report.  Need answer so this is more clear.
+
+This table also lists the matching assets by manufacturer and includes the following additional information: model number, serial number, date contract expires, operating system, number of days left in contract, date installed and vendor phone number.
+
+This report can be customized with a 3 company logos that are approximately 195px by 50px.  One logo is in the report header, another in the page header and a third in the page footer.
 
 == Parameters
 
-[options="header"]
-[cols="1,3"]
+[options="header, autowidth"]
+[cols="1,2,3,4"]
 
 |===
 | Field
+| Type
 | Description
+| Default
 
-| Show expired maintenance contracts
-| Specify true or false.
+| Show Expired Maintenance Contracts
+| String
+| Specify true or false.  True will include contracts that have already expired along with assets with maintenance contracts that will expire within the number of days specified for the warning threshold below.
+| true
 
-| Format for expire date
-| Default: YYYY-MM-DD
+| Format for Expire Date
+| String
+| Specify the format for the expiration date.
+| YYYY-MM-DD
+// QUESTION: Not sure why this is here.  Shouldn't the expiration date be stored as a date field?  If so, format is not necessary, unless it is for output.  If it is an output format preference, it should be a more generic parameter.  If all the dates in the maintenance contracts have different formats, then this will be problematic.
```

Comment:
Probably, though all assets are technically string values

Reply:
So the dates are stored in Postgres as strings, not dates?

Reply:
Depends on the field. The expiration date asset field is a string.
![image](https://user-images.githubusercontent.com/26419758/168182103-b7934999-8d1c-4ef3-8e3f-ddc8efb72bf6.png)